__tests__/app.test.js

```diff
@@ -83,3 +83,25 @@ describe('/api/articles/:article_id', () => {
       });
   });
 });
+describe('/api/articles', () => {
+  test('GET:200 sends articles to the client', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toBeInstanceOf(Array);
+        expect(body.articles.length > 0).toBe(true);
+        body.articles.forEach((article) =>{
+          expect(article).toBeInstanceOf(Object);
```

Comment:
Have a look at jest matchers here to make the test more robust, rather than just checking the type of the value. 

.toHaveProperty, .toMatchObject or expect.any can be helpful